packages/istanbul-lib-source-maps/test/transformer.test.js

```diff
@@ -80,4 +74,29 @@ describe('transformer', function() {
             }
         });
     });
+
+    it('maps each file only once, /path/to/file.js and \\path\\to\\file.js are the same file', function() {
+        if (isWindows()) {
+            return this.skip();
+        }
+
+        var coverageMap = createMap({}),
+            testDataSlash = createData(),
```

Comment:
Ideally we could just have `const testDataSlash = Object.assign({}, coverageData, { stuff: 'we need for this test' })` and so on.

Reply:
Updated now, the tests are a lot easier to read now imho ☺️

Reply:
🙌